Front-end/Auction_App_Front/app/components/new-arrivals.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import {inject as service} from '@ember/service';
+
+export default Component.extend({
+  productHttp: service(),
+  pageNumber: '0',
+  arrivals: null,
+  init() {
+    this._super(...arguments);
+    this.get('productHttp').loadArrivals(this.get('pageNumber')).then((result) => {
+      this.set('arrivals', result);
+    })
+  },
+  actions: {
+    loadMore() {
+      var temp = +this.get('pageNumber') + 1;
+      this.set('pageNumber', temp.toString());
+      this.get('productHttp').loadArrivals(this.get('pageNumber')).then((result) => {
+        this.set('arrivals', result);
```

Comment:
this just replaces the content, if you are using infinite load you should add the next page to the existing content